### PR TITLE
Removes the SWAT bepis node, moves its contents to a new cargo pack, except for the baton

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
The baton goes to the illegal tech node, as it's quite strong.
The crate is called "SWAT Tactical Operations crate", and much like the SWAT crate, it contains 2 sets of tactical equipment, but no weapons

## Why It's Good For The Game
The bepis node was too easily acquirable

## Changelog
:cl:
add: Added the black baton to illegal technology node
add: Added a tactical SWAT cargo pack, with the advanced vest, night visor helmet and other tactical equipment
del: Removed SWAT bepis node
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/cargo/packs/bloat.dm [hunk 1]
```diff
@@ -21,6 +21,22 @@
...
 	crate_name = "swat taser crate"
 
+/datum/supply_pack/security/armory/swat_tactical
+	name = "SWAT Tactical Operations crate"
+	desc = "Contains two sets of lightweight equipment for tactical operations. Each set contains a vest, night vision helmet, mask, combat belt, and combat gloves. Requires Armory access to open."
+	cost = 6000
+	contains = list(/obj/item/clothing/head/helmet/advanced,
+					/obj/item/clothing/head/helmet/advanced,
+					/obj/item/clothing/suit/armor/vest/advanced,
+					/obj/item/clothing/suit/armor/vest/advanced,
```
Comment (BobJoga) [inline modular_skyrat/code/modules/cargo/packs/bloat.dm:31]: should be the tech armor instead IMHO

Comment (BobJoga) [reply modular_skyrat/code/modules/cargo/packs/bloat.dm:31]: or maybe make swat tech-armor a separate crate

Comment (Azarak) [reply modular_skyrat/code/modules/cargo/packs/bloat.dm:31]: I dont know if fullbody armor is suitable for lightweight equipment, besides, this was supposed to be a replacement for that node

Comment (BobJoga) [reply modular_skyrat/code/modules/cargo/packs/bloat.dm:31]: I mean - Fair enough tbh, but i do feel that tech armor MK2 should be accessible somehow, i do think a separate crate with another name can work.

File: modular_skyrat/code/modules/cargo/packs/bloat.dm [hunk 2]
```diff
@@ -21,6 +21,22 @@
 					/obj/item/gun/energy/e_gun/stun)
 	crate_name = "swat taser crate"
 
+/datum/supply_pack/security/armory/swat_tactical
+	name = "SWAT Tactical Operations crate"
+	desc = "Contains two sets of lightweight equipment for tactical operations. Each set contains a vest, night vision helmet, mask, combat belt, and combat gloves. Requires Armory access to open."
+	cost = 6000
+	contains = list(/obj/item/clothing/head/helmet/advanced,
```
Comment (ShadeAware) [inline modular_skyrat/code/modules/cargo/packs/bloat.dm:28]: Wheres the Mark II Techarmor?

Comment (Azarak) [reply modular_skyrat/code/modules/cargo/packs/bloat.dm:28]: It's lightweight swat gear, not tech armor

Comment (ShadeAware) [reply modular_skyrat/code/modules/cargo/packs/bloat.dm:28]: The tech armor was included in the node. Include the Mark II Techarmor in this crate aswell.

**Review by KathrinBailey — APPROVED** [review]
Woah, the sensible branch name is putting me to shame with my silly branch names.
All checks out.

**Pull request merged**